app/models/jsonld_record.rb

```diff
@@ -92,7 +93,8 @@ def title_language
   def metadata_map
     {
       author_display:        'creator',
-      description_display:   'description',
+      call_number_browse_s:  'call_number',
```

Comment:
That record has holdings at multiple locations, each with the same call number. However, there's no utility to having duplicate values there (I think there was at one point and the call number needed to display twice). If you get rid of the [allow_duplicates option](https://github.com/pulibrary/marc_liberation/blob/development/marc_to_solr/lib/traject_config.rb#L967) for the field, it should work fine.